docs/reference/components/transforms/reduce.cue

```diff
@@ -135,7 +135,90 @@ components: transforms: reduce: {
 
 	examples: [
 		{
-			title: "Reduce Rails Logs"
+			title: "Merge Ruby exceptions"
+			input: [
+				{
+					log: {
+						timestamp: "2020-10-07T12:33:21.223543Z",
+						message: "foobar.rb:6:in `/': divided by 0 (ZeroDivisionError)",
+						host: "host-1.hostname.com"
+						pid: 1234
+						tid: 5678
+					}
+				},
+				{
+					log: {
+						timestamp: "2020-10-07T12:33:21.223543Z",
+						message: "    from foobar.rb:6:in `bar'",
+						host: "host-1.hostname.com"
+						pid: 1234
+						tid: 5678
+					}
+				},
+				{
+					log: {
+						timestamp: "2020-10-07T12:33:21.223543Z",
+						message: "    from foobar.rb:2:in `foo'",
+						host: "host-1.hostname.com"
+						pid: 1234
+						tid: 5678
+					}
+				},
+				{
+					log: {
+						timestamp: "2020-10-07T12:33:21.223543Z",
+						message: "    from foobar.rb:9:in `<main>'",
+						host: "host-1.hostname.com"
+						pid: 1234
+						tid: 5678
+					}
+				},
+				{
+					log: {
+						timestamp: "2020-10-07T12:33:22.123528Z",
+						message: "Hello world, I am a new log",
+						host: "host-1.hostname.com"
+						pid: 1234
+						tid: 5678
+					}
+				},
+			]
+			configuration: {
+				group_by: ["host", "pid", "tid"]
+				marge_strategies: message: "concat_newline"
+				starts_when: {
+					type: "remap"
+					source: 'match(.message, /^[^\s]/)'
```

Comment:
Does this `\s` need an extra `\` in the Cue source?

Reply:
I don't think so. It was necessary when using non-literal strings in TOML, hence the `'` characters.